doc/producers/custom.md

```diff
@@ -136,4 +136,15 @@ we get a result like this :
 }
 ```
 
-(For this to actually work we would need to add resolvers to the User object to resolve the `id` and `name` properties).
+For this to actually work we would need to add resolvers to the User object to resolve the `id` and `name` properties like so:
+```php
+    $registry->addFieldResolver('User', 'id', $builder->callback(function ($account) {
```

Comment:
any reason for the indentation here? We don't have it in the field resolver examples further above.